setup.py

```diff
@@ -121,7 +121,7 @@ def read(*names, **kwargs):
         'open source', 'scan', 'license', 'package', 'dependency',
         'copyright', 'filetype', 'author', 'extract', 'licensing',
     ],
-    python_requires='>=2.7,<3',    
+    # python_requires='>=2.7,<3',    
```

Comment:
After porting, we have to drop the support for python 2.Minimum version of python will be 3.6.1 as suggested by you.